examples/main.go

```diff
@@ -5,20 +5,28 @@ import (
 	"log"
 
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/resource"
 
 	"github.com/honeycombio/otel-config-go/otelconfig"
 )
 
 func main() {
-	otelShutdown, err := otelconfig.ConfigureOpenTelemetry()
+	otelShutdown, err := otelconfig.ConfigureOpenTelemetry(
+		otelconfig.WithResourceOption(
+			resource.WithAttributes(
+				attribute.String("resource.example_set_in_code", "CODE"),
+				attribute.String("resource.example_clobber", "CODE_WON"),
+			),
+		),
+	)
 
 	if err != nil {
 		log.Fatalf("error setting up OTel SDK - %e", err)
 	}
 
 	defer otelShutdown()
 	tracer := otel.Tracer("my-app")
-	ctx := context.Background()
-	ctx, span := tracer.Start(ctx, "doing-things")
+	_, span := tracer.Start(context.Background(), "doing-things")
```

Comment:
I … don't know why I included this. It has nothing to do with the resource attributes issue. 😐